File: UNIDAD 3/JAVA/U3_P17.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.u3_p17;
import java.util.Scanner;
/**
 *
 * @author LaboratorioLTI2
 */
public class U3_P17 {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        char opcion;
        do {
        System.out.println("Bienvenido a una sola ejecucion ");
        System.out.println("¡¡¡Escribe n para salir!!! ");
        opcion =sc.next().charAt(0);
        } while (opcion != 'n');
    }
}

File: UNIDAD 3/C#/U3_P17.cs
using System;
namespace U3_P17
{
    internal class U3_P17
    {
        static void Main(string[] args)
        {
            char opcion;
            do
            {
                Console.WriteLine("Bienvenido a una sola ejecucion ");
                Console.WriteLine("¡¡¡Escribe n para salir!!! ");
                opcion = Console.ReadKey().KeyChar;
            } while (opcion != 'n');



        }
    }
}